collect_app/src/main/java/org/odk/collect/android/activities/DataManagerList.java

```diff
@@ -195,7 +195,7 @@ private void createDeleteInstancesDialog() {
         mAlertDialog = new AlertDialog.Builder(this).create();
         mAlertDialog.setTitle(getString(R.string.delete_file));
         mAlertDialog.setMessage(getString(R.string.delete_confirm,
-                mSelected.size()));
+                String.valueOf(mSelected.size())));
```

Comment:
It's a lint requirement. I'm hypothesizing here because I can't find the documentation for it but I believe it's to avoid the extra call?